src/main/java/net/minecraftforge/common/ForgeConfigSpec.java

```diff
@@ -60,23 +61,20 @@
 import com.google.common.collect.Lists;
 import com.google.common.collect.ObjectArrays;
 
-/*
+/**
  * Like {@link com.electronwill.nightconfig.core.ConfigSpec} except in builder format, and extended to accept comments, language keys,
  * and other things Forge configs would find useful.
  */
-public class ForgeConfigSpec extends UnmodifiableConfigWrapper<UnmodifiableConfig> //TODO: Remove extends and pipe everything through getSpec/getValues?
+public class ForgeConfigSpec extends UnmodifiableConfigWrapper<UnmodifiableCommentedConfig> //TODO: Remove extends and pipe everything through getSpec/getValues?
 {
-    private Map<List<String>, String> levelComments = new HashMap<>();
```

Comment:
About translations for categories: We could add a map for cat path -> translation key, just like we had with `levelComments` before

Reply:
My thoughts as well